docs/getting_started/basic_usage/basic_usage.rst

```diff
@@ -27,58 +27,26 @@ as well as some training parameters.
 Some common config parameters:
 
 task
-    The task, such as ``train`` or ``forward``.
+    The task, such as ``train``, ``forward`` or ``search``.
 
 device
     E.g. ``gpu`` or ``cpu``. Can also be ``gpu0,gpu1`` for multi-GPU training.
```

Comment:
There should also be a newline.
And better or maybe just remove the comment for multi-GPU training. This is for Theano.

Reply:
Note that this is an option not necessarily needed. It would automatically detect the device. By explicitly defining it, you just enforce it (and it would raise an exception if it cannot get what you requested).